src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -48,7 +48,9 @@ public function load(array $configs, ContainerBuilder $container): void
 
         if ($config['debug']['enabled'] || (!empty($config['cache_control']))) {
             $debugHeader = $config['debug']['enabled'] ? $config['debug']['header'] : false;
+            $ttlHeader = $config['cache_control']['ttl_header'] ?? '';
```

Comment:
the array value will always be set, so no need for the `??` i think.
```suggestion
            $ttlHeader = $config['cache_control']['ttl_header'];
```

Reply:
oki lets remove completely then